### ISSUE TITLE
Incorporate Aspace "Container Type" into enum construction

### ISSUE THREAD

**Issue by DaltonAlves** [gwu-libraries/aspace-barcodes#4]
Just noticed an error with our matching that I should have caught earlier. We aren't considering the `TC container type` in the Aspace enum for matching. I'm going to attach some screenshots to demonstrate what is happening now, without the `TC container type` being considered. Essentially, indicators are re-used/duplicated for different container types which is causing us to match on the incorrect Alma Item record. Ex. MS1111 Volume 1 is matching on MS1111 Box 1.

There's probably a few different ways we could approach this. We could consider the container type in our matching logic by including it in the Aspace enum that we construct. Or, we could exclude any TC that is not a "Box" container type. Both approaches would require us to update the Aspace db query to pull container type, I think.

This is our current confirguation of constructing the Aspace enum: 
-as the TC indicator
-as the series-level (archival object) component_id plus the TC indicator

This is what I'm proposing:
-as the **TC container type** plus TC indicator
-as the series-level (archival object) component_id plus the **TC container type** plus the TC indicator

I assume we need to alter something on the Alma side of the logic too, possibly the box pattern? 

![2](https://github.com/gwu-libraries/aspace-barcodes/assets/110255670/06db63f6-e85f-4ac2-8d41-6dd85d8dab14)

![1 (1)](https://github.com/gwu-libraries/aspace-barcodes/assets/110255670/5587e06d-012d-404e-b34f-4f87368b3bd5)

**Comment by dolsysmith** [comment]
Thanks, @DaltonAlves! This is a good catch. 

Regarding the two solutions you proposed, are the non-box container types ever represented in the Enum field in Alma?

**Comment by DaltonAlves** [comment]
Yes, the box type is _usually_ represented in the enum a field. Here's an example of an MMS ID with different container types represented in the enums of the items: 99118784703604107.
